execute/executetest/transformation.go

```diff
@@ -31,30 +31,36 @@ const ulp uint = 2
 // Comparison options for floating point values.
 // NaNs are considered equal, and float64s must
 // be sufficiently close to be considered equal.
-var floatOptions = cmp.Options{
-	cmpopts.EquateNaNs(),
-	cmp.FilterValues(func(x, y float64) bool {
-		return !math.IsNaN(x) && !math.IsNaN(y)
-	}, cmp.Comparer(func(x, y float64) bool {
-		// If sufficiently close, then move on.
-		// This avoids situations close to zero.
-		if scalar.EqualWithinAbs(x, y, tol) {
+func floatOptions(tol float64) cmp.Options {
+	if tol == 0 {
+		tol = defaultTol
+	}
+	return cmp.Options{
+		cmpopts.EquateNaNs(),
+		cmp.FilterValues(func(x, y float64) bool {
+			return !math.IsNaN(x) && !math.IsNaN(y)
+		}, cmp.Comparer(func(x, y float64) bool {
+			// If sufficiently close, then move on.
+			// This avoids situations close to zero.
+			if scalar.EqualWithinAbs(x, y, tol) {
+				return true
+			}
+			// If not sufficiently close, both floats
+			// must be within ulp steps of each other.
+			if !scalar.EqualWithinULP(x, y, ulp) {
+				return false
+			}
 			return true
-		}
-		// If not sufficiently close, both floats
-		// must be within ulp steps of each other.
-		if !scalar.EqualWithinULP(x, y, ulp) {
-			return false
-		}
-		return true
-	})),
+		})),
+	}
 }
 
 func ProcessTestHelper(
 	t *testing.T,
 	data []flux.Table,
 	want []*Table,
 	wantErr error,
+	epsilon float64,
 	create func(d execute.Dataset, c execute.TableBuilderCache) execute.Transformation,
 ) {
```

Comment:
One approach I've seen taken in situations like this one is to change the signature:
```suggestion
	create func(d execute.Dataset, c execute.TableBuilderCache) execute.Transformation,
	...cmp.Option,
) {
```
And then pass any additional options at just the callsites where they're needed. This way, none of the unaffected tests need to change, and by passing a `cmp.Option` it can have a name that is clear rather than a `0` which can be opaque.

What do you think?